### PR TITLE
feat: Update release flow

### PR DESCRIPTION
**Pull request by dgusakov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,3 +66,15 @@ $ python -m myproject
 
 Hello world!
 ```
+
+## Release flow
+
+To create new release:
+
+1. Merge all changes to the `main` branch
+1. Navigate to Repo => Actions
```
Comment (F4ever) [inline README.md:75]: Increment numbers

Comment (dgusakov) [reply README.md:75]: No need. MD handles it correctly

File: .github/workflows/prepare-release.yml [hunk 2]
```diff
@@ -16,8 +16,8 @@ jobs:
           git-message: "chore(release): {version}"
           tag-prefix: ""
           output-file: "CHANGELOG.md"
-          version-file: "pyproject.toml"
-          version-path: "tool.poetry.version"
+          version-file: "package.json"
+          version-path: "version"
```
Comment (avsetsin) [inline .github/workflows/prepare-release.yml:20]: doesn't seem right, is a python template

Comment (dgusakov) [reply .github/workflows/prepare-release.yml]: I thought I fixed it... Fixed now!

**Review by avsetsin — APPROVED** [review]

**Pull request merged**